CONTRIBUTING.md

```diff
@@ -65,36 +78,22 @@ $ juju deploy --trust \
     --resource prometheus-image=ubuntu/prometheus:latest
 ```
 
-## Linting
-Flake8 and black linters may be run to check charm and test source code using the
-command
 
-```bash
-tox -e lint
-```
-
-## Testing
-
-Unit tests are implemented using the Operator Framework test
-[harness](https://ops.readthedocs.io/en/latest/#module-ops.testing). These
-tests may executed by doing
-
-```bash
-$ tox -e unit
-```
-
-It is expected that unit tests should provide at least 80% code coverage.
 
 ## Code Overview
```

Comment:
I wounder if we should entirely drop the "code overview" section.

Reply:
yes, it seems to me that it does not add much value...